torch/distributed/tensor/_dispatch.py

```diff
@@ -144,7 +145,6 @@ def dispatch(
 
         # extract local tensor and sharding infos to a OpInfo
         op_info = self.unwrap_to_op_info(op_call, args, kwargs)
-        logger.debug("Dispatching op_call: %s", op_info.schema)
```

Comment:
These should not have that much overhead, do they really?

Reply:
> do they really

My measurements are noisy but per the summary it looked to me like they cost 5-10 usec per dispatch. timeit says the cost for a single trivial disabled log is most of a microsecond (result is total seconds to run the benchmark and it's 1 million iterations by default), and this was multiple logs that also pass arguments.

```
>>> timeit.timeit(stmt="logger.debug('hello')", setup="import logging; logger = logging.getLogger('test')")
0.8429189168382436
```